### DISCUSSION TITLE
Unnable to reverify my student status

### DISCUSSION THREAD

**Discussion by lihuencarranza** [community/community#123640] [GitHub Education]
### Hi everyone,

I'm having trouble reverifying my student status.

When I registered last year I didn't have any problem. I have a certification that I'm a regular student in my university with the date of emission. 

This is the message that I'm receiving:

_Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on May 13, 2024 for @lihuencarranza._
- _Please secure your GitHub account with two-factor authentication. You may need to log out and log back in to GitHub before reapplying._
- **Please use your device camera to submit academic affiliation documents.**
- _Please ensure that your academic affiliation document contains your first name exactly as it appears in your GitHub billing information. Variations of your name and nicknames are unacceptable._
- _You are unlikely to be verified until you have completed your GitHub user profile with your full name exactly as it appears in your academic affiliation document. Please do not use a variation of your name or a nickname. Once you have updated your profile information log out and log back into GitHub before re-applying._
_Once you have resolved the issues with your sumbission, visit us again and reapply._ 

I've already check everything, I don't know what else to do.

**Comment by nikhilsbuilds** [comment]
It seems there are a few potential issues that are preventing the verification of your student status. Here are some suggestions to help resolve this:

1. **Enable Two-Factor Authentication (2FA)**: Enabling 2FA on your GitHub account adds an extra layer of security, which may be required for the student verification process. Follow GitHub's instructions to set up 2FA using an authenticator app or SMS.

2. **Submit Documents Using Camera**: The message specifically asks you to use your device's camera to capture and submit your academic affiliation documents. This likely means uploading a clear photo or scan of your student ID, enrollment certificate, or other official document from your university.

3. **Match Your Name Exactly**: Ensure that your first name on your GitHub profile matches exactly how it appears on your academic affiliation document. Even small variations or nicknames may cause issues. Update your GitHub profile with your full legal name as it appears on your student ID or enrollment documents.

4. **Log Out and Log Back In**: After making any changes to your GitHub profile or enabling 2FA, log out of your GitHub account and log back in. This can sometimes refresh the system and allow the changes to take effect.

5. **Check Document Validity**: Make sure that the academic affiliation document you are submitting is current and valid. Some institutions issue new documents each academic year or term, so an outdated document may not be accepted.

If you have already taken these steps and are still experiencing issues, you may want to reach out to GitHub's support team for further assistance. They may be able to provide more specific guidance or identify any other potential issues with your submission.
